_posts/2023-08-23-even-friendlier-sql.md

```diff
@@ -0,0 +1,574 @@
+---
+
+layout: post
+title:  "Even Friendlier SQL with DuckDB"
+author: Alex Monahan
+excerpt_separator: <!--more-->
```

Comment:
Pretty sure this has to be used somewhere in the doc to take effect?

Reply:
Yup, you are correct. That was just lying around. It's a remnant from when we had previews on our blog list page. I can remove it!